Makefile

```diff
@@ -34,12 +34,17 @@ init:
 	pip install pipenv --upgrade
 	pipenv run pip install --upgrade pip
 	pipenv install
-	cp {Pipfile,Pipfile.lock,setup.py} ${BUILD_CONTEXT}
+	pwd
```

Comment:
Looks good overall. Assuming `pwd` and `ls` command is added for debugging purpose in future right?

Reply:
oh my mistake, I should remove that. Thanks for pointing out.